Model/src/GamePackage/Die.java

```diff
@@ -0,0 +1,53 @@
+package GamePackage;
+
+import java.util.Random;
+
+/*****************************************************************
+ The Die Class will be a random number generator that "rolls"
+ a die some methods to return the value of the die.
+
+ @author Dylan Kernohan
+ @version Winter 2018
+ *****************************************************************/
+public class Die {
+
+    /** current value of the die */
+    int value;
+
+    /*****************************************************************
+     Default constructor
+     *****************************************************************/
+    public Die( ) {

```

Comment:
The constructor should instantiate the value . Any value you chose between 1- 6 should be fine.

Reply:
We do not want the constructor to instantiate a value. When we create the dice, it does not need a value until we call role() honestly could get rid of constructor, but its good practice to write it in, even if empty.